internal/operators/manager.go

```diff
@@ -463,13 +448,25 @@ func EnsureLVMAndCNVDoNotClash(cluster *common.Cluster, openshiftVersion string,
 		}
 	}
 
-	if lvmEnabled && cnvEnabled {
+	if !lvmEnabled || !cnvEnabled {
+		return nil
+	}
+
+	// Openshift version greater or Equal to 4.12.0 support cnv and lvms
+	if isGreaterOrEqual, _ := common.BaseVersionGreaterOrEqual(lvm.LvmsMinOpenshiftVersion4_12, openshiftVersion); !isGreaterOrEqual {
 		return errors.Errorf("Currently, you can not install OpenShift Data Foundation Logical Volume Manager operator at the same time as Virtualization operator.")
 	}
 
-	if lvmEnabled && !common.IsSingleNodeCluster(cluster) {
-		return errors.Errorf("OpenShift Data Foundation Logical Volume Manager operator is only supported for Single Node Openshift")
+	if isGreaterOrEqual, _ := common.BaseVersionGreaterOrEqual(lvm.LvmMinMultiNodeSupportVersion, openshiftVersion); isGreaterOrEqual {
+		return nil
+	}
+
+	// before 4.15 multi node LVM not supported
```

Comment:
I think that this condition should be before 
`if isGreaterOrEqual, _ := common.BaseVersionGreaterOrEqual(lvm.LvmMinMultiNodeSupportVersion, openshiftVersion); isGreaterOrEqual {`
let's see if your unit test will pass